participants/2021/datenanfragen.yml

```diff
@@ -1,8 +1,8 @@
----
-Name: Datenanfragen.de e. V.
-Website: https://www.datarequests.org
-Swag:
-  - shirt
-  - stickers
-Description: "The first 100 participants who submit a pull request or a patch that is accepted by us to one of the qualifying repositories between October 1, 2020 and November 1, 2020, will get a free sticker set. In addition, we are awarding 10 t-shirts for the best contributions. Our board will select those winners after the event is over."
-Details: https://www.datarequests.org/blog/hacktoberfest-2020/
+---
+Name: Datenanfragen.de e. V.
+Website: https://www.datarequests.org
+Swag:
+  - shirt
+  - stickers
+Description: "The first 100 participants who submit a pull request or a patch that is accepted by us to one of the qualifying repositories between October 1, 2020 and November 1, 2020, will get a free sticker set. In addition, we are awarding 10 t-shirts for the best contributions. Our board will select those winners after the event is over."
+Details: https://www.datarequests.org/blog/hacktoberfest-2020/

```

Comment:
Was it meant to be https://www.datarequests.org/blog/hacktoberfest-2021/?

Reply:
Yeah, i forgot to press the "Save" button, before pushing 😸